tools/deb/envoy.json

```diff
@@ -1,13 +1,12 @@
 {
-  "listeners": [
-  ],
+  "listeners": [],
   "lds": {
     "cluster": "lds",
-    "refresh_delay_ms": 10000
+    "refresh_delay_ms": 1000
```

Comment:
why so low/often?

Reply:
Will fix it in another PR, not critical for now ( the file will be replaced with v2 api soon)